pkg/container/container.go

```diff
@@ -157,6 +162,19 @@ func (c *containerModule) Run(ns string, data pkg.Container) (id pkg.ContainerID
 		return id, ErrEmptyRootFS
 	}
 
+	startupFile, err := os.Open(filepath.Join(data.RootFS, ".startup.toml"))
```

Comment:
Can this be moved to another method, it looks like a lot of code in the Run method . Something like `updateEnv(&data, path.Join(root, ".startup.toml"))`